### ISSUE TITLE
Load only certain tissues from GTEx

### ISSUE THREAD

**Issue by nuno-agostinho** [nuno-agostinho/psichomics#298]
Present a filter for tissues after loading GTEx data.

**Comment by nuno-agostinho** [comment]
```R
library(data.table)
# Check available tissues and respective samples
samples <- fread("~/GTEx/GTEx_Data_V6_Annotations_SampleAttributesDS.txt",
                 data.table=FALSE, check.names=TRUE)
tissues <- unique(samples$SMTS)
tissues

# Retrieve ID from breast samples
tissue <- "Breast"
samples <- samples$SAMPID[samples$SMTS == tissue]

# Check which columns refer to the previous sample ID
cols <- fread("~/GTEx/GTEx_junction_reads.txt", data.table=FALSE, nrows=0)
cols <- names(cols) %in% samples
cols <- ifelse(cols, "integer", "NULL")
cols[1] <- "character" # The junction identifier is required

# Benchmark: only read columns of interest vs. read all columns
library(microbenchmark)

junctionFile <- "~/GTEx/GTEx_junction_reads.txt"
nrows <- 5000
microbenchmark(
    times=5,
    "Some columns"=fread(junctionFile, data.table=FALSE, nrows=nrows,
                         colClasses=cols),
    "All columns"=fread(junctionFile, data.table=FALSE, nrows=nrows))

# Unit: seconds
#          expr      min       lq     mean   median       uq     max neval
#  Some columns 2.483689 2.499744 2.497651 2.501331 2.501380 2.50211     5
#   All columns 5.307432 5.444168 5.492252 5.446282 5.581749 5.68163     5

```

**Comment by nuno-agostinho** [comment]
Closed by 752408a8
